Simplenote/Simplenote-Info-Hockey.plist

```diff
@@ -53,7 +53,7 @@
 	<key>NSPrincipalClass</key>
 	<string>SPApplication</string>
 	<key>SUFeedURL</key>
-	<string>https://rink.hockeyapp.net/api/2/apps/ede9c49346946c51a1db1db167b49892</string>
+	<string>https://api.appcenter.ms/v0.1/public/sparkle/apps/ede9c493-4694-6c51-a1db-1db167b49892</string>
```

Comment:
See #639. Without this, the update check would fail with 

```
[Sparkle] Error: An error occurred in retrieving update information. Please try again later. An error occurred while parsing the update feed. (URL (null))
[Sparkle] Error: An error occurred while parsing the update feed. (null) (URL (null))
[Sparkle] Error: Line 6: Opening and ending tag mismatch: hr line 5 and body
 (null) (URL (null))
```